tests/test_etl.py

```diff
@@ -1,5 +1,57 @@
+import os
+import pytest
+import shutil
+import subprocess
+
+
 def test_hello_world():
     """
     Test the tests.
     """
     assert "Hello World" == "Hello World"
+
+def data_exists(data_folder):
+    """
+    data_folder
+    """
+    path = os.path.join("tests", "data", data_folder)
+    return os.path.exists(path) and os.listdir(path)
+
+
+@pytest.mark.skipif(not data_exists("mimic-iv-demo"), reason="Data not available, skipping tests")
+class TestMimicETL:
+    """
+    Test the MIMIC ETL.
+    """
+    @classmethod
+    def setup_class(cls):
+        """
+        Setup method that runs before any tests in the class.
+        Used here to run the ETL process.
+        """
+        cls.source_path = os.path.join("tests", "data", "mimic-iv-demo")
+        cls.destination_path = os.path.join(cls.source_path, "build")
+
+        # Remove the build directory if it exists
+        if os.path.exists(cls.destination_path):
+            shutil.rmtree(cls.destination_path)
+
+        # Run the ETL
+        subprocess.run(['meds_etl_mimic', cls.source_path, cls.destination_path, "--num_shards", "10"], check=True)
```

Comment:
We probably want to test that the files actually load with HuggingFace datasets.

dataset = datasets.Dataset.from_parquet(cls.destination_path + '/data/*')

example_patient = dataset[0]

assert 'patient_id' in example_patient
assert 'events' in example_patient

Reply:
Good idea, I was really just going for the bare minimum in this pull request (i.e. do the build run and did it generate files) but I'll add your load step now. One minute...